scss/_utilities.scss

```diff
@@ -528,8 +528,8 @@ $utilities: map-merge(
       class: lh,
       values: (
         1: 1,
-        sm: $line-height-sm,
-        base: $line-height-base,
+        sm: 1.25,
```

Comment:
Is there a reason why the values are now hard coded instead of using variables?

What happen to:
```
$line-height-base:            calc(18 / 16) !default;
$line-height-sm:              calc(16 / 14) !default;
$line-height-lg:              calc(30 / 16) !default;
````
in `_variables.scss`? Should we remove them?

+ Correct me if I am wrong but the values for `$line-height-sm` and `$line-height-base` in `_variables.scss` and `_variables.scss` will now longer be the same?

+ Missing Boosted mod?